From 95e0fa6b0499e5eafb18ba1a04272549d4f6da87 Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Sat, 4 Feb 2017 16:18:38 +0800
Subject: [PATCH 079/249] cm: Remove android-visualizer library

This was originally imported for the music player app.
However, the library was replaced with an improved visualizer
a while back, making this library unused now.

This essentially reverts commit e4da816f4c547331fcc8c36a5413ca2d2a55c780.

Change-Id: I8ec6bb1f640b860a75ebecdc7ce3b11b085fcac4
---
 snippets/cm.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/cm.xml b/snippets/cm.xml
index 160f480..89fa7e5 100644
--- a/snippets/cm.xml
+++ b/snippets/cm.xml
@@ -3,7 +3,6 @@
 
   <!-- LineageOS additions -->
   <project path="android" name="LineageOS/android" />
-  <project path="external/android-visualizer" name="LineageOS/android_external_android-visualizer" />
   <project path="external/bash" name="LineageOS/android_external_bash" />
   <project path="external/cmsdk-api-coverage" name="LineageOS/android_external_cmsdk-api-coverage" />
   <project path="external/exfat" name="LineageOS/android_external_exfat" />
-- 
2.17.1

